config/manager/manager.yaml

```diff
@@ -59,17 +59,131 @@ spec:
         #   type: RuntimeDefault
       containers:
       - command:
-        - /manager
-#        args:
+          - /manager
+        args:
+          - run
 #          - --leader-elect
 #          - --health-probe-bind-address=:8081
         image: controller:latest
         name: manager
+        env:
+          - name: IMAGE_UPDATER_INTERVAL
+            valueFrom:
+              configMapKeyRef:
+                key: interval
+                name: image-updater-config
+                optional: true
+          - name: IMAGE_UPDATER_LOGLEVEL
+            valueFrom:
+              configMapKeyRef:
+                name: image-updater-config
+                key: log.level
+                optional: true
+          - name: MAX_CONCURRENT_APPS
+            valueFrom:
+              configMapKeyRef:
+                name: image-updater-config
+                key: max_concurrent_apps
+                optional: true
+          - name: MAX_CONCURRENT_RECONCILES
+            valueFrom:
+              configMapKeyRef:
+                name: image-updater-config
+                key: max_concurrent_reconciles
+                optional: true
+          - name: GIT_COMMIT_USER
+            valueFrom:
+              configMapKeyRef:
+                name: image-updater-config
+                key: git.user
+                optional: true
+          - name: GIT_COMMIT_EMAIL
+            valueFrom:
+              configMapKeyRef:
+                name: image-updater-config
+                key: git.email
+                optional: true
+          - name: GIT_COMMIT_SIGNING_KEY
+            valueFrom:
+              configMapKeyRef:
+                key: git.commit-signing-key
+                name: image-updater-config
+                optional: true
+          - name: GIT_COMMIT_SIGNING_METHOD
+            valueFrom:
+              configMapKeyRef:
+                key: git.commit-signing-method
+                name: image-updater-config
+                optional: true
+          - name: GIT_COMMIT_SIGN_OFF
+            valueFrom:
+              configMapKeyRef:
+                key: git.commit-sign-off
+                name: image-updater-config
+                optional: true
+          - name: IMAGE_UPDATER_KUBE_EVENTS
+            valueFrom:
+              configMapKeyRef:
+                name: image-updater-config
+                key: kube.events
+                optional: true
+          - name: ARGOCD_LOGLEVEL
```

Comment:
Not related to this PR but `ARGOCD_LOGLEVEL` seems legacy var. It's not used anywhere. I think it can be safely removed from all places.

Reply:
Good catch. I'll remove them.